tests/test_wps_climdex_ptot.py

```diff
@@ -6,17 +6,16 @@
 
 
 def buil_params(climdex_input, ci_name, threshold, vector_name, output_file):
-    datainputs = (
+    params = (
```

Comment:
Why change from `datainputs` to `params`?

Reply:
just to stay consistent with the other ones I wrote. I also spelt 'built' wrong :facepalm: